ui/lib/core/addon/components/secret-list-header.js

```diff
@@ -7,15 +7,16 @@ import Component from '@glimmer/component';
 
 /**
  * @module SecretListHeader
- * SecretListHeader component is breadcrumb, title with icon and menu with tabs component. Hello
+ * SecretListHeader component is breadcrumb, title with icon and menu with tabs component.
  *
  * Example is wrapped in back ticks because this component relies on routing and cannot render an isolated sample, so just rendering template sample
  * @example
  * ```
- * <SecretListHeader @isCertTab={{eq this.tab "cert"}} @model={{this.model}} @baseKey={{this.baseKey}} @backendCrumb={{this.backendCrumb}} @filter={{this.filter}} />
+ * <SecretListHeader @model={{this.model}} />
  * ```
  *
  * @param {object} model - Model used to pull information about icon and title and backend type for navigation.
+ * @param {boolean} [isConfigure=false] - Boolean to determine if the configure tab should be shown.
  */
 
 export default class SecretListHeader extends Component {
```

Comment:
Up to you, but if this component has truly been stripped down as much as it has then I don't see a need for the jsdoc and backing class. It seems like the `isKv` getter logic could easily be moved to the template: 

```
{{#if includes @model.engineType (array "kv" "generic")}}  
```

Reply:
Oh, great call. Let me double check this flow and will amend if it works out.

Reply:
Welp, the passed in isConfigure is a little harder to remove. Leaving for now.

Reply:
How does that passed arg related to the backing class?